src/components/CustomerQuota/CustomerQuotaScreen.tsx

```diff
@@ -178,27 +178,14 @@ const CanBuyResult: FunctionComponent<{
         <View style={styles.resultWrapper}>
           {Object.entries(cart)
             .sort((itemOne, itemTwo) => {
-              const itemOneName = getProduct(itemOne[0])?.name || "";
-              const itemOneOrder = itemOneName.slice(
-                0,
-                itemOneName.indexOf(" ")
-              );
-              const itemTwoName = getProduct(itemTwo[0])?.name || "";
-              const itemTwoOrder = itemTwoName.slice(
-                0,
-                itemTwoName.indexOf(" ")
-              );
+              const ProductOneOrder = getProduct(itemOne[0])?.order || 0;
```

Comment:
May want to use camelCase here (:

Reply:
Ohh, ooops, thanks for the check. Will edit!

Reply:
Done :D